sdk/tdf_config.go

```diff
@@ -186,6 +186,16 @@ func WithSegmentSize(size int64) TDFOption {
 	}
 }
 
+// WithAssertions  returns an Option that add assertions to TDF.
+func WithAssertions(assertionList ...Assertion) TDFOption {
+	return func(c *TDFConfig) error {
+		newAssertions := make([]Assertion, 0)
+		newAssertions = append(newAssertions, assertionList...)
+		c.assertions = newAssertions
+		return nil
+	}
+}
```

Comment:
Should we define an `AssertionConfig` struct so only the fields we want set are allowed to be. For example should a user be allowed to pass in their own binding?

Reply:
I'm not sure about binding. They can pass in RSA key for signing and verification. I agree we should have AssertionConfig so we can pass Keys.